### PR TITLE
Added Water Seven 17

### PR DESCRIPTION
**Pull request by trulow**
## Checklist for submitting new .nfo files
- [x] I have removed the old .nfo files that are being replaced
- [x] I have validated that the new .nfo file works correctly in Plex
- [x] I have updated the README, fixing the list of current missing episodes
- [x] I have modified exceptions.json, removing any exceptions no longer required

### CODE REVIEW

**Comment by cawnj** [comment]
@trulow Thanks for the change!

With the nfo file, the aired date and duration in seconds are incorrect.

The aired date should be the release date on onepace.net - for this ep it's `Released on: 2/17/2024`
And for the duration in seconds, run the file through tinyMediaManager to get the correct file/media info (this makes sure all the info like audio tracks, subtitle languages, etc. are correct - but also the duration). For this ep the value should be `1509`

Everything else though looks perfect to me!

**Comment by trulow** [comment]
Updated and also added episode 18 as well

File: One Pace/Season 18/One Pace - S18E17 - Sogeking.nfo [hunk 1]
```diff
@@ -1,25 +1,29 @@
...
+
+Manga Chapter(s): 366-367
+
+Anime Episode(s): 256-257</plot>
   <runtime>0</runtime>
   <mpaa/>
-  <premiered/>
-  <aired/>
+  <premiered>2024-02-16</premiered>
+  <aired>2024-02-16</aired>
```
Comment (cawnj) [inline One Pace/Season 18/One Pace - S18E17 - Sogeking.nfo:22]: This should be `2024-02-17` as displayed on [onepace.net/watch](https://www.onepace.net/watch) in the episode description

**Review by cawnj — CHANGES_REQUESTED** [review]
Added one comment regarding E17. Everything else looks good, and E18 also looks good. Ty @trulow 🙏

**Comment by trulow** [comment]
Dated updated

**Review by cawnj — APPROVED** [review]
Ty, LGTM!

**Pull request merged**